lib/cisco_node_utils/bgp_neighbor_af.rb

```diff
@@ -223,7 +229,8 @@ def additional_paths_receive=(val)
     end
 
     def default_additional_paths_receive
-      config_get_default('bgp_neighbor_af', 'additional_paths_receive').to_sym
+      ret = config_get_default('bgp_neighbor_af', 'additional_paths_receive')
+      ret.to_sym unless ret.to_s == ''
```

Comment:
I thought I had a nil vs '' issue and was following: http://stackoverflow.com/questions/247948/is-there-a-better-way-of-checking-nil-or-length-0-of-a-string-in-ruby

But I replaced it with nil? and it's passing so not sure what I was seeing.